apis_configs/indexd_settings.py

```diff
@@ -61,4 +61,6 @@ def load_json(file_name):
         )
     )
 
+CONFIG["PRESIGNED_URL_ENDPT"] = "http://presigned-url-fence-service/"
+
```

Comment:
that CONFIG variable should default to that value
we don't like to update those settings.py files - we want to retire those
it looks like kube-setup-indexd does update that secret automatically: https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-indexd.sh#L22
but I hate that we have .py code in cloud-automation/ that gets inserted into service runtimes
try to setup the service configuration so that it loads all non-secret configuration from configmaps setup out of the manifest folder
and all secret information comes from g3auto/ secrets